app/utils/searchUtils.js

```diff
@@ -18,9 +18,12 @@ function getFetchParamsFromFilters(filters) {
       filters.duration
     ),
     { purpose: filters.purpose === 'all' ? '' : filters.purpose },
-    { page: filters.page || 1 }
+    { page: filters.page || 1 },
+    { order_by: filters.orderBy || '' }
```

Comment:
should have 1 default filter active instead of ``

Reply:
Made simple refactor for this in new commit. Did not include order by filter at all if no orderBy value is present. I hope this is fine.